data/remnant/remnant missions.txt

```diff
@@ -4634,12 +4628,412 @@ mission "Remnant: Expanded Horizons Storms 1"
+	waypoint
+		system "Caeculus"
+	waypoint
+		system "Ildaria"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Storms 1: done"
+		random < 30
+	on offer
+		require "Research Laboratory"
+		conversation
+			`	You have barely started your meal at a cafe table when you notice Dawn waving at you from another table. You walk over and sit down to see what she needs. "Hello, Captain <last>! Would you be interested in going on another research expedition?"`
+			choice
+				`	"Of course! Where do you need to go?"`
+				`	"Sorry, I'm busy right now."`
+					defer
+
+			`	"This study will actually require an expedition to two different stars, but the first is quite close by: Caeculus. We may have to remain there for some time, but what we observe should be quite exciting.`
+			choice
+				`	"What exactly will we be observing?"`
+					goto massejection
+				`	"Exciting? I'm not sure I like the sound of that."`
+
+			`	"Oh, it will not be dangerous if we keep our distance. This should be far safer than the expedition to Sagittarius A*, and for this first star we'll be right next door to the capital in case of any emergencies. Afterward, we will need to journey to Ildaria, in Ancestral space, so help should likewise be available."`
+			choice
+				`	"Okay, you've convinced me. What will we be watching, anyway?"`
+					goto massejection
+
+			label massejection
+			`	Her eyes light up as she animately begins explaining "As you've probably noticed by now, the Ember Waste is filled with dead and dying stars, relics of an age long past. However, Remnant scientists have been puzzled for centuries by the fact that the age of these stars does not match the astrophysical models we brought over from Ancestral space. Based on those models, these stars are all far too young to produce novae, or too small to go supernova. So, we want to compare two stars at similar states in their stellar evolution: one inside the Ember Waste, and one in the wider galaxy. Because they usually only live for a few million years, and are physically distinct, we have chosen two Wolf-Rayet stars to serve as our baseline."`
+			`	"In order to most accurately record their internal structures, we want to observe a coronal mass ejection event for each one. With luck, this should be enough to establish the accuracy of our models, or perhaps point towards some hidden phenomenon present in the Ember Waste to explain the disparity. "`
+			`	"I can explain more during the flight, if you wish. You do still have a research laboratory installed, right?"`
+			choice
+				`	"Yes, of course!"`
+
+			`	You follow Dawn out of the cafeteria and to the spaceport, where you prepare for departure.`
+				accept
+
+	on enter Caeculus
+		conversation
+			`	As soon as you arrive, Dawn quickly sets up the laboratory, setting up her instruments before joining you in the cockpit. "It may take some time for a coronal mass ejection event to occur, so we may be here for a while."`
```

Comment:
```suggestion
			`	As soon as you arrive, Dawn quickly sets up the laboratory, setting up her instruments before joining you in the cockpit. "It may take some time for a coronal mass ejection event to occur, so we may be here for a while.`
```